phoonnx_train/engines/vits.py

```
@@ -0,0 +1,216 @@
+            sample_rate=config.sample_rate,
+            dataset=[str(p) for p in dataset_paths],
+            **config.extra,
+            **kwargs,
+        )
+
+    def export_onnx(
+        self,
+        checkpoint_path: Path,
+        config_path: Path,
+        output_dir: Path,
+        **kwargs: Any,
```

Ruff (0.15.1)

[warning] Unused method argument: `kwargs`

(ARG002)